tools/rna_tools/remurna/remurna.xml

```diff
@@ -0,0 +1,124 @@
+<tool id="remurna2" name="remuRNA" version="1.0.0">
+    <description> Measurement of Single Nucleotide Polymorphism induced Changes of RNA Conformation </description>
+    <requirements>
+                <requirement type="package" version="1.2">rnasnp</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" />
+    </stdio>
+    <command><![CDATA[
+    touch input.fa &&
+    python $__tool_directory__/create_input_file.py '$seq' '$snp' input.fa &&
+    \$(which remuRNA)
```

Comment:
Unfortunately, this is the only way this tool can work. There is a condition inside the source code that checks if a '/' exits, and the way to do it is to use 'which'.